al1s/Lab02-Unit-Testing
# Add ATM interactive screens

What is done:

- [X] Create a bank “ATM” machine within a console application with VS2017
- [X] Your solution should include the following:
  - Main method
  - Testable external methods
  - User interface that prompts the user for standard ATM operations
    - View Balance
    - Withdraw Money
    - Add Money (not standard, but for this exercise it will be….) =	
    - Keep asking the user to choose a transaction until they choose to ‘exit’ the application
  - Exception Handling. Implement try, catch, finally and a throw within your program.
- [X] Expected behavior:
  - Make sure the user can’t go below a zero balance
  - Make sure your methods are just returning values and not reading input from the console.
  - When running your application, it should activate the user interface without any direct code manipulation
